.github/workflows/pull-request.yml

```
@@ -0,0 +1,54 @@
+          node-version: '20.x'
+      - name: Cache npm
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: npm ci
+        working-directory: sep490-frontend
+      - run: npm run lint
+        working-directory: sep490-frontend
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)